pkg/handlers/session.go

```diff
@@ -47,21 +47,27 @@ func parseClusterAuthorization(authHeader string) (authorization, error) {
 	}, nil
 }
 
-func requireValidSession(kotsStore store.Store, w http.ResponseWriter, r *http.Request) (*sessiontypes.Session, error) {
+func requireValidSession(kotsStore store.Store, w http.ResponseWriter, r *http.Request) (sess *sessiontypes.Session, err error) {
```

Comment:
i don't see why naming the response parameters here is necessary

Reply:
Updated in https://github.com/replicatedhq/kots/pull/3846